critiquebrainz/ws/review/views.py

```diff
@@ -416,6 +416,7 @@ def review_post_handler(user):
     """
 
     def fetch_params():
+        global REVIEW_TEXT_MIN_LENGTH
```

Comment:
This fix looks incorrect to me. We should not be reassigning a global variable (especially which we want to asssume to be constant) while processing a request as it will be reflected in future requests. For example, a draft review may set to REVIEW_TEXT_MIN_LENGTH, then another review comes which is not a draft, since REVIEW_TEXT_MIN_LENGTH is None the minimum length check will not work as expected.